nixos/modules/system/boot/systemd/tmpfiles.nix

```diff
@@ -20,6 +20,59 @@ in
       '';
     };
 
+    systemd.tmpfiles.settings = mkOption {
+      description = ''
```

Comment:
`description` values should follow proper sentence structure and grammar

additionally... if we're going to the effort of making a really nice wrapper like this (❤️ @Lassulus) we should really make a good description that explains why NixOS has a special relationship with `tmpfiles` - our love of declarative approach over `bash` scripts - and specifically emphasize that `tmpfiles` is a misleading name

i mention this because the name really throws off new people! outside of NixOS this feature of `systemd` is not as popular and well known so we should really clarify the situation... then people can say "that nixos distro has really good docs!" 😄

Reply:
I wanted to extend the description after we are happy with the interface :)

Reply:
Just extended it (with @lassulus permission)!

Reply:
Just extended it (with @lassulus permission)!